update-versions.sh

```diff
@@ -24,8 +23,6 @@ PKG_NAMES="@streamr/utils
 @streamr/protocol
 @streamr/proto-rpc
 @streamr/dht
```

Comment:
The `trackerless-network` should also follow the unified versioning, and should therefore be included here?